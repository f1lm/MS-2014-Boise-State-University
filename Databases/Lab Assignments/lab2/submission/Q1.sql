-- 1.  Retrieve the names of all students who major in computer science.
select FName, LName from STUDENT s INNER JOIN MAJOR m ON s.SSN = m.SSN And m.DeptCode =
(select DeptCode from DEPT Where DName='Computer Science') order by FName, LName;