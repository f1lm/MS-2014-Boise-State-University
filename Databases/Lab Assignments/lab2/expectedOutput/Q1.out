mysql> source Q1.sql
+-------+---------+
| FName | LName   |
+-------+---------+
| John  | Smith   |
| Susan | Johnson |
| Alex  | Mace    |
| Karen | Geer    |
| Jason | Freeman |
+-------+---------+
5 rows in set (0.00 sec)

mysql> exit
